<launch>
  <node name="gps_serial_decoder" pkg="slam" type="gps_serial_decoder" output="screen"/>
  <node name="imu_decoder" pkg="slam" type="imu_decoder" output="screen"/>

  <node name="get_pitch" pkg="slam" type="imu_getter" output="screen"/>
  <node name="kalman_fusion" pkg="slam" type="kalman_fusion" output="screen"/>

  <node name="lidar_decoder" pkg="slam" type="lidar_decoder"/>

  <rosparam file="$(find slam)/slam_para.yaml"/>
</launch>
